File: src/main/java/dev/rivasjf/expensemanager/Controller/MainController.java
package dev.rivasjf.expensemanager.Controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.time.OffsetDateTime;

@RestController
public class MainController {

    @GetMapping()
    public String helloWorld() {
        return "Hello World!";
    }

    @GetMapping("/now")
    public String now() {
        return OffsetDateTime.now().toString();
    }
}

File: src/main/java/dev/rivasjf/jessysecurity/controller/MainController.kt
package dev.rivasjf.jessysecurity.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import kotlin.time.Clock

@RestController
class MainController {

    @GetMapping()
    fun home(): String {
        return "Hello, World!"
    }

    @GetMapping("/now")
    fun now(): String {
        return Clock.System.now().toString()
    }
}